ooni/nettest.py

```diff
@@ -591,11 +594,16 @@ def generateMeasurements(self):
                 test_instance._setUp()
                 test_instance.summary = self.summary
                 for method in test_methods:
+                    try:
+                        measurement = self.makeMeasurement(
+                            test_instance,
+                            method,
+                            test_input)
+                    except Exception:
+                        log.exception(failure.Failure())
```

Comment:
I think it's better if you pass to the log.exception the actual exception so that we log also the exception type together with the traceback.

ex.

```
except Exception as exc:
    log.exception(exc)
```